pkg/util/provider/drain/drain_test.go

```diff
@@ -1146,3 +1284,40 @@ func updateVolumeAttachments(drainOptions *Options, pvName string, nodeName stri
 
 	drainOptions.volumeAttachmentHandler.AddVolumeAttachment(newVolumeAttachment)
 }
+
+// matchPodPersistentVolumeNames applies the given matcher to the result of PodVolumeInfo.PersistentVolumeNames().
+func matchPodPersistentVolumeNames(matcher gomegatypes.GomegaMatcher) gomegatypes.GomegaMatcher {
+	return gcustom.MakeMatcher(func(actual PodVolumeInfo) (bool, error) {
+		return matcher.Match(actual.PersistentVolumeNames())
+	})
+}
+
+// matchPodVolumeIDs applies the given matcher to the result of PodVolumeInfo.VolumeIDs().
+func matchPodVolumeIDs(matcher gomegatypes.GomegaMatcher) gomegatypes.GomegaMatcher {
+	return gcustom.MakeMatcher(func(actual PodVolumeInfo) (bool, error) {
+		return matcher.Match(actual.VolumeIDs())
+	})
+}
+
+func addAll(informer cache.SharedIndexInformer, objects ...any) {
```

Comment:
consider making this function generic, then you don't need to cast to any in all callers and introduce methods like `toListOfAny`, but cast only in the implementation of this function.